csi/crypto/crypto.go

```diff
@@ -69,7 +71,10 @@ func (cp *EncryptParams) GetPBKDF() string {
 }
 
 // VolumeMapper returns the path for mapped encrypted device.
-func VolumeMapper(volume string) string {
+func VolumeMapper(volume, dataEngine string) string {
+	if dataEngine == string(longhorn.DataEngineTypeV2) {
+		return path.Join(mapperFilePathPrefix, volume+mapperV2VolumeSuffix)
```

Comment:
Reuse `getEncryptVolumeName()`
```suggestion
		return path.Join(mapperFilePathPrefix, getEncryptVolumeName(volume))
```

Reply:
Modified. Thanks.